core/src/main/java/hudson/model/ListView.java

```diff
@@ -167,16 +169,24 @@ public boolean hasJobFilterExtensions() {
     public DescribableList<ListViewColumn, Descriptor<ListViewColumn>> getColumns() {
         return columns;
     }
-    
+
+
+     public List<TopLevelItem> getItems() {
```

Comment:
What is this for? Does this need `@Override`, `@Restricted` or `@since`?

Reply:
Yep, lost `@Override` without explanation.

Reply:
I will cleanup the Javadoc / Override changes after the merge